tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/WorkspaceMigrationImpl.java

```diff
@@ -119,10 +119,14 @@ public void problemsFound( List<Problem> problems )
                 try
                 {
                     final TaskProblem taskProblem = new TaskProblem( problem, false );
-                    final IMarker marker =
-                        workspaceResource.createMarker( MigrationConstants.MIGRATION_MARKER_TYPE );
+                    //remove problems in WEB-INF/classes folder
+                    if( !workspaceResource.getFullPath().toString().contains( "WEB-INF/classes" ) )
```

Comment:
Please introduce a new method called "shouldAddMarker" that will check for paths that contain values that we want to avoid and add it to the if(){} block